packages/@react-types/provider/src/index.d.ts

```diff
@@ -15,6 +15,11 @@ import {ReactNode} from 'react';
 
 export type ColorScheme = 'light' | 'dark';
 export type Scale = 'medium' | 'large';
+export interface Breakpoints {
+  S: number,
+  M: number,
+  L: number
+}
```

Comment:
Should we also support custom breakpoints? e.g. if a user prefers to name them `phone`/`tablet`/`desktop` or have an `XL` breakpoint etc. Maybe add an index signature here as well?

```ts
interface Breakpoints {
   S: number,
   M: number,
   L: number,
   [custom: string]: number
 }
```